lib/github-pages-health-check/domain.rb

```diff
@@ -192,7 +192,10 @@ def github_domain?
 
       # Is the host our Fastly CNAME?
       def fastly?
-        !!host.match(/\Agithub\.map\.fastly\.net\.?\z/i)
+        !!(
+          host.match(/\Agithub\.map\.fastly\.net\.?\z/i) ||
+          host.match(/\Asni\.github\.map\.fastly\.net\.?\z/i)
```

Comment:
Wondering if it's cleaner to have a `FASTLY_HOSTS` array constant, and then just use `host.match /\A#{Regexp.union(FASTLY_HOSTS)}\z/` to dry things up.